src/emcpy/plots/tools.py

```diff
@@ -0,0 +1,94 @@
+class ColorbarPreset:
+    """
+    Class that has pre-determined vmin, vmax,
+    and cmap values depending on the variable
+    and evaluation type.
+    """
+    def __init__(self, variable, eval_type):
+        """
+        Class constructor.
+
+        Args:
+            variable : (str) variable of interest
+            eval_type : (str) evaluation type
+        """
+
+        cbar_d = {
+            'brightness temperature': {
+                'diff': {
+                    'vmax': -5,
+                    'vmin': 5,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmax': 320,
+                    'vmin': 200,
+                    'cmap': 'rainbow'
+                }
+            },
+            'specific humidity': {
+                'diff': {
+                    'vmin': 0.005,
+                    'vmax': 0.005,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmin': 0,
+                    'vmax': 0.025,
+                    'cmap': 'YlGnBu'
+                }
+            },
+            'temperature': {
+                'diff': {
+                    'vmax': -5,
+                    'vmin': 5,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmax': 320,
+                    'vmin': 200,
+                    'cmap': 'rainbow'}
+            },
+            'wind speed': {
+                'diff': {
+                    'vmax': -5,
+                    'vmin': 5,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmax': 75,
+                    'vmin': 0,
+                    'cmap': 'viridis'
+                },
+            },
+            'wind': {
+                'diff': {
+                    'vmax': -5,
+                    'vmin': 5,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmax': 50,
+                    'vmin': -50,
+                    'cmap': 'viridis'
+                }
+            }
+        }
+
+        variable = 'wind' if variable in ['uwind', 'vwind', 'ugrd', 'vgrd'] \
+                   else variable
+
+        try:
+            self.vmin = cbar_d[variable][eval_type]['vmin']
+            self.vmax = cbar_d[variable][eval_type]['vmax']
+            self.cbar = cbar_d[variable][eval_type]['cmap']
+
+        except KeyError:
+            if variable not in cbar_d.keys():
+                raise TypeError(f'{variable} is not a valid variable.' +
```

Comment:
what about a default value to return here for everything? Or is the assumption that the user should not call this if they want default?

Reply:
There is already default values set in the plotting classes. The assumption should be that this is only to be called if they want preset values. Regardless, you will have to set the plotting object's vmin, vmax, and colormap equal to these values i.e.
```
scatterobj = MapScatter(lat, lon, data)
cbar_presets = ColorbarPreset('temperature', 'diff')
scatterobj.vmin = cbar_presets.vmin
scatterobj.vmax = cbar_presets.vmax
scatterobj.cmap = cbar_presets.cmap
```

Reply:
Ah okay got it, so you would presumably do something like:
```
if var in ['temperature', 'brightness temperature']: # etcetera
    cbar_presets = ColorbarPreset(var, 'diff')
    scatterobj.vmin = cbar_presets.vmin
    scatterobj.vmax = cbar_presets.vmax
    scatterobj.cmap = cbar_presets.cmap
```

I guess I'm asking is this the best way to do it? I'm not sure.